src/runner.sh

```diff
@@ -109,7 +109,8 @@ if ((BASH_VERSINFO[0] >= 4)); then
   )
 
   runner_colorize() {
-    echo "${runner_colors[$1]}${*:2}${runner_colors[reset]}"
+    message=("${@:2}")
+    (IFS=" "; echo "${runner_colors[$1]}${message[*]}${runner_colors[reset]}")
```

Comment:
Creating a subshell in a hot function like this would slow everything down by a lot. Perhaps this works.

```suggestion
    IFS=" " echo "${runner_colors[$1]}${message[*]}${runner_colors[reset]}"
```

Reply:
Thanks for pointing this out. I didn't realize but there is some impact indeed.

* Original `task_test_colorize` - about 33s
* Proposed `task_test_colorize` (with subshell) - about 44s

Note: times were roughly measured in 100 runs, including Helm command. Nothing extra precise, only to get an idea about an impact.

Setting IFS for a single command doesn't work (see https://stackoverflow.com/questions/25153582/why-cant-i-set-ifs-for-one-call-of-printf and https://www.shellcheck.net/wiki/SC2097).

I slightly modified your suggested change by saving IFS to a temporary value and setting it back at the end of the function.

* Latest `task_test_colorize` (saving IFS) - about 33s (change impact is not measurable)

Reply:
Looks good.